### PR TITLE
Test structure to JSON parsing added

### PR DESCRIPTION
**Pull request by mateuszkaleta**
Implementation is located in TestRunnerJSON.jl file and is included in package module
The tests are located in TestRunnerJSON.tests.jl file which is included in runtests.jl

The logic behind is quite simple: the test tree nodes are first converted to dictionaries and then parsed by external library (JSON.jl ,https://github.com/JuliaLang/JSON.jl ).

### CODE REVIEW

**Comment by gdziadkiewicz** [comment]
Great, I will merge it in a moment

File: src/TestRunnerJSON.jl [hunk 1]
```diff
@@ -0,0 +1,38 @@
+using JSON
+
+function json(testStructureNodes::Array{TestRunner.TestStructureNode})
+  treeDictionary = [to_dict(node) for node in testStructureNodes]
+  JSON.json(treeDictionary)
+end
+
+function to_dict(factGroup::FactsNode)
```
Comment (gdziadkiewicz) [inline src/TestRunnerJSON.jl]: @mateuszkaleta , can you change it to FactsCollectionNode? That's the new name for FactsNode.

**Comment by gdziadkiewicz** [comment]
@mateuszkaleta, I did first merge my branch and there are conflicts now :( Can you refresh your branch with master? I believe that only important change is that FactsNode was renamed to FactsCollectionNode.

**Comment by mateuszkaleta** [comment]
Everything should be fine now :)

**Pull request merged**